app/views/canonical_pending_transactions/_canonical_pending_transaction.html.erb

```diff
@@ -124,7 +124,11 @@
     </div>
   </td>
   <td class="nowrap text-right">
-    <%= render_transaction_amount pt.amount %>
+    <% if pt.amount_pending? %>
+      PENDING
+    <% else %>
+      <%= render_transaction_amount pt.amount %>
+    <% end %>
```

Comment:
See earlier comment about pending amounts - I really don't think we should do this.